package.json

```diff
@@ -25,7 +25,7 @@
     "cz": "cz",
     "clean": "lerna clean",
     "build": "lerna run build",
-    "build:storybook": "build-storybook -s ./public",
```

Comment:
I'm sorry, I missed the build:storybook script 🤦‍♂️

Reply:
No worries it happens 🤗